packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -293,5 +293,29 @@ export const comments: { readonly [key in TokenName]: string } = {
     mentionFontFamily: 'Font family for mention views',
     mentionFontSize: 'Font size for mention views',
     mentionFontWeight: 'Font weight for mention views',
-    mentionFontLineHeight: 'Font line height for mention views'
+    mentionFontLineHeight: 'Font line height for mention views',
+    calendarStaticEventBackgroundColor:
+        'Background color for static calendar events',
+    calendarDynamicEventBackgroundColor:
+        'Background color for dynamic calendar events',
+    calendarTransientEventBackgroundColor:
+        'Background color for transient calendar events',
+    calendarStaticEventBorderColor: 'Border color for static calendar events',
+    calendarTransientEventBorderColor:
+        'Border color for transient calendar events',
+    calendarStaticEventFontColor: 'Font color for static calendar events',
+    calendarDynamicEventFontColor: 'Font color for dynamic calendar events',
+    calendarTransientEventFontColor: 'Font color for transient calendar events',
+    calendarStaticEventHoverColor:
+        'Color while hovering static calendar events',
+    calendarDynamicEventHoverColor:
+        'Color while hovering dynamic calendar events',
+    calendarTransientEventHoverColor:
+        'Color while hovering transient calendar events',
+    calendarResourceSelectionBackgroundColor:
+        'Background color while calendar resource is selected',
+    calendarBlockedStrokeColor:
+        'Background fill to indicate the occupied slots in calendar resource rows',
+    calendarEventResizeRingColor:
```

Comment:
It's not clear to me what visual this is associated with. Can you provide a screenshot?

Reply:
![image](https://github.com/user-attachments/assets/808fd894-77b9-45ee-8b1d-186ceeee78d6)